src/components/shared/Button/CommandBarButton.css

```diff
@@ -33,3 +33,7 @@
   display: inline-block;
   vertical-align: middle;
 }
+
+.command-bar-button .togglePanes {
```

Comment:
Introducing this makes the top and bottom padding uneven. I'd rather avoid it if possible - can this be dropped with the fix working as intended?

Reply:
`vertical-align: middle;` will not force top and bottom paddings to be even, I added vertical-align in pixels.